jekyll/_cci2/pipelines.adoc

```diff
@@ -0,0 +1,146 @@
+
+==== Add a configuration source
+
+TIP: If you created a project xref:create-project#[in the CircleCI web app] you will find a configuration source is already set up.
+
+image::{{site.baseurl}}/assets/img/docs/add-configuration-source.png[Add a configuration source]
+
+To add a configuration source, follow these steps:
+
+. In the CircleCI web app, click on **Projects** in the sidebar
+. Click on the ellipsis next to your project (icon:ellipsis-h[]) and then click **Project Settings**
+. Select **Configuration** in the sidebar
+. Click btn:[Add Configuration Source]
+. Complete the form fields and options:
+** Give your configuration source a descriptive name, for example `Run tests and deploy`
+** Under Configuration Source select the platform that matches where your project is set up (for example, GitHub App)
++
+NOTE: For GitLab Self-managed, you can input any instance that you have previously set up in CircleCI. If you wish to set a different feature branch or repository from a self-managed instance as a new configuration source, you will first need to add a new connection via your xref:gitlab-integration#organization-settings-integrations[menu:Organization Settings[Integrations]]. In either case, you will also need to enter your personal access token again to authorize this connection.
+** Authorize your connection if this is not already showing with a icon:check-circle[]
+** Select your repository
++
+CAUTION: **Select the same repository for your configuration source and trigger** (described in the following section). There is currently a bug that means that CircleCI will always look for the configuration source in the same repository from which the triggering event was emitted.
+** (Optional) If you will be assigning an inbound webhook trigger to this configuration source, select a fallback branch to use in case the triggering event does not have a triggering branch.
+** Enter the path to your configuration file relative to the root of your repository. For example, `.circleci/config.yml`. If you are integrated through the GitHub App, you can enter any path to a `.yml` file, and the file can have any name you choose.
+. Click btn:[Save]
+
+Once you have set up a configuration source you need to set up a project trigger to connect it to. This is described in the following section.
+
+=== Project triggers
+
+TIP: If you created a project xref:create-project#[in the CircleCI web app] you will find a trigger is already set up.
+
+Triggers are rules that define which configuration source should be run on a given event. A project can have one or more triggers, which are associated with events from a source of change. Triggers include, but are not limited to, a push to a branch on a version control system (VCS), for example, a GitHub repository. A trigger determines an event (or events) along with a configuration source to define a pipeline. A trigger can have only one configuration source but a configuration source can have multiple triggers.
```

Comment:
> A trigger determines an event (or events) along with a configuration source to define a pipeline.
I get the meaning but this reads a little strange.

Reply:
I think I'll just take that part out, it's covered in the rest of the text